src/logger.ts

```diff
@@ -11,75 +16,87 @@ const LEVEL = Object.freeze({
 });
 
 export default class Logger {
-  constructor (options = {}) {
-    this.formatter = options.formatter || Logger.colorFormatter;
-    this.destination = options.destination || process.stderr;
+  formatter: LogFormatter;
+  destination: NodeJS.WritableStream;
+  private _level: number;
+  history: LogEvent[];
+  private readonly _historySize: number;
+  constructor (options: {
```

Comment:
Missing blank line before constructor. And we don't really mark underscore properties as `private` or `readonly`.